tests/test_jira_client.py

```diff
@@ -0,0 +1,99 @@
+from unittest.mock import patch
+
+import pytest
+
+from retasc.jira_client import JiraClient
+
+
+@pytest.fixture
+def mock_jira():
```

Comment:
Can we mock the underlying requests instead (use `requests_mock` fixture)? This would ensure that the Jira library accepts all the parameters and does makes the expected REST API calls.